src/content/docs/en/guides/deploy/zerops.mdx

```diff
@@ -0,0 +1,178 @@
+    adapter: node({
+      mode: "standalone"
+    }),
+  });
+```
+
+## Building and deploying your app to Zerops
+
+Now that you you've set up your app and prepared a Node.js service on Zerops, we can trigger the build and deploy pipeline on Zerops.
+
+Start by describing how to build and run your app through `zerops.yml` file placed at the root of your codebase. `setup:` part should match the hostname your chose for your service, in case of our example - `hellothere`.
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```yaml
+      zerops:
+        - setup: hellothere
+          build:
+            base: nodejs@20
+            buildCommands:
+              - npm i
+              - npm run build
+            deploy:
+              - dist
+              - package.json
+              - node_modules
+            cache:
+              - node_modules
+              - package-lock.json
+          run:
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```yaml
+      zerops:
+        - setup: hellothere
+          build:
+            base: nodejs@20
+            buildCommands:
+              - pnpm i
+              - pnpm run build
+            deploy:
+              - dist
+              - package.json
+              - node_modules
+            cache:
+              - node_modules
+              - pnpm-lock.json
+          run:
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```yaml
+      zerops:
+        - setup: astronode
+          build:
+            base: nodejs@20
+            buildCommands:
+              - yarn
+              - yarn build
+            deploy:
+              - dist
+              - package.json
+              - node_modules
+            cache:
+              - node_modules
+              - yarn.lock
+          run:
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+### Trigger the pipeline by connecting the service to GitHub / GitLab
+To setup continuous deployment on either push to branch or a new release, go to your Node.js service detail and connect your Zerops service with either GitHub or GitLab repository.
+
+
+### Trigger the pipeline Using Zerops CLI (zcli)
+
+1. Install Zerops CLI
+```shell
+# To download the zcli binary directly,
+# use https://github.com/zeropsio/zcli/releases
+npm i -g @zerops/zcli
+```
+
+2. Open [`Settigs > Access Token Management`](https://app.zerops.io/settings/token-management) in Zerops app, generate a new access token
+
+3. Login
+```shell
+zcli login <token>
+```
+
+4. Navigate to your apps root (where zerops.yml is also placed)
+```shell
+zcli push
+```
```

Comment:
```suggestion
<Steps>
1. Install the Zerops CLI.
		```shell
		# To download the zcli binary directly,
		# use https://github.com/zeropsio/zcli/releases
		npm i -g @zerops/zcli
		```

2. Open [`Settings > Access Token Management`](https://app.zerops.io/settings/token-management) in the Zerops app and generate a new access token.

3. Log in using your access token with the following command:
		```shell
		zcli login <token>
		```

4. Navigate to the root of your app (where `zerops.yml` is located) and run the following command to trigger the deploy:
		```shell
		zcli push
		```
</Steps>
```
Fixing indentation so code blocks align with the numbers, and using our `<Steps>` component!